net/mwan3/files/usr/sbin/mwan3track

```diff
@@ -86,13 +86,21 @@ main() {
 	config_get failure_interval $1 failure_interval $interval
 	config_get_bool keep_failure_interval $1 keep_failure_interval 0
 	config_get recovery_interval $1 recovery_interval $interval
+	config_get_bool check_quality $1 check_quality 0
+	config_get max_latency $1 max_latency $max_latency
+	config_get min_latency $1 min_latency $min_latency
+	config_get max_loss $1 max_loss $max_loss
+	config_get min_loss $1 min_loss $min_loss
```

Comment:
Could we also load the defaults if no max/min values are found in the config?

Reply:
Let's try to identify some sane / safe defaults.
For packetloss, <5% and >20% should work.
For latency, it is difficult as monitor IP (openDNS or google DNS) latency would differ across the globe, unless user checks the base latency and configures mwan3 properly. 

<500ms and >1000ms should be safe, although not sane.

Your views?